tests/test_utilities.py

```diff
@@ -778,3 +778,22 @@ def test_has_duplicates():
 
     with pytest.raises(ValueError):
         raise_has_duplicates(np.array([0, 1, 2, 2]))
+
+
+def test_copy_vtk_array():
+    with pytest.raises(TypeError, match='Invalid type'):
+        pyvista.utilities.misc.copy_vtk_array([1, 2, 3])
+
+    value = 10
+    arr = vtk.vtkFloatArray()
+    arr.SetNumberOfValues(2)
+    arr.SetValue(0, value)
+    arr_copy = pyvista.utilities.misc.copy_vtk_array(arr, deep=True)
+    arr_copy.GetValue(0)
```

Comment:
This call seems to "work" even when no value is set. So perhaps this line should be replaced with
```suggestion
    assert arr_copy.GetNumberOfValues()
```
if the failure mode it expects is "the copied array is blank". Or just removed otherwise.